app/code/Transoft/Blog/view/frontend/templates/blogListTmpl.phtml

```diff
@@ -0,0 +1,24 @@
+<?php
+use Transoft\Blog\ViewModel\Bloglist;
+
+/** @var \Transoft\Blog\Block\Bloglist $block*/
+/** @var Bloglist $bloglistViewModel */
+
+$bloglistViewModel = $block->getData('bloglist_view_model');
+$data = $bloglistViewModel->getPostCollection()->getData();
+$match = $bloglistViewModel->isTypesMatching();
+
+?>
+<?php if ($match): ?>
+    <div>
+        <h3><?= /* @noEscape */ __('Latest 5 blogs:') ?></h3>
+        <ul>
+            <?php foreach ($data as $value):
+                $item = $value['theme'];
+                $url = $block->escapeUrl($bloglistViewModel->getUrlById($value['blog_id']));
+                ?>
+                <?=  /* @noEscape */ "<li><a href='$url'>$item</a></li>" ?>
```

Comment:
Please use HTML markup instead of string output

Reply:
fixed